### PR TITLE
Add Kotlin code generation for proto types

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR adds Kotlin code generation for proto types of this project.
Some tests were also migrate to Kotlin, using the generated Kotlin DSL.

The PR also addresses some validation issues that were revealed when using latest project dependencies.

### CODE REVIEW

File: core/src/test/kotlin/io/spine/core/AcksTest.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+@DisplayName("`Acks` utility class should")
+internal class AcksTest : UtilityClassTest<Acks>(Acks::class.java) {
+
+    fun newAck(signalId: SignalId): Ack = ack {
+        messageId = signalId.pack()
+        status = statusOk()
+    }
+
+    @Nested
+    internal inner class `obtain 'CommandId'`() {
```
[the remark this answers is not in the document]
Comment (armiol) [reply core/src/test/kotlin/io/spine/core/AcksTest.kt:47]: @alexander-yevsyukov looks there could be a point in this comment.

**Review by armiol — APPROVED** [review]
LGTM with a few comments to address.

File: core/src/test/kotlin/io/spine/core/AcksTest.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply core/src/test/kotlin/io/spine/core/AcksTest.kt:47]: There are two methods in this test class.

File: buildSrc/src/main/kotlin/io/spine/internal/gradle/javac/ErrorProne.kt [hunk 2]
```diff
@@ -82,7 +82,6 @@ private object ErrorProneConfig {
 
             "-Xep:CheckReturnValue:OFF",
             "-Xep:FloggerSplitLogStatement:OFF",
-            "-Xep:FloggerLogString:OFF"
```
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply buildSrc/src/main/kotlin/io/spine/internal/gradle/javac/ErrorProne.kt:85]: Sorry. I did it again :)

**Pull request merged**